modules/eco-setting-node-maintenance-cli.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-setting-node-maintenance-cli_{context}"]
+= Setting a node to maintenance mode in the CLI
```

Comment:
"Setting a node to maintenance mode" -> "Setting a node into maintenance mode" ?

Same as for "Setting a node to maintenance mode with a NodeMaintenance custom resource" occurrence